uktrade/data-hub-frontend
# Redirect to the matching confirmation page after a DNB company is clicked

## Description of change

Redirect user to the matching confirmation page after a D&B company is clicked from the search results list.

## Test instructions

1. Go to the company match page `companies/COMPANY_ID/match`
2. Search for a company
3. Click one of the results
4. Check if you are on the company match confirmation page
